builder/package.json

```diff
@@ -38,25 +38,23 @@
     "@semantic-release/git": "9.0.0",
     "@semantic-release/npm": "7.0.3",
     "@semantic-release/release-notes-generator": "9.0.0",
-    "semantic-release": "17.0.4",
-    "@angular-devkit/architect": "0.900.3",
-    "@angular-devkit/schematics": "9.0.3",
     "@types/glob": "7.1.1",
-    "@types/jest": "24.9.1",
+    "@types/jest": "^25.1.3",
     "@types/mime": "2.0.1",
     "@types/mime-types": "2.1.0",
     "@types/node": "11.15.7",
-    "glob": "7.1.6",
-    "jest": "24.9.0",
+    "cpy-cli": "^3.1.0",
+    "jest": "^25.1.0",
     "json-schema-to-typescript": "8.1.0",
+    "semantic-release": "17.0.4",
     "ts-jest": "25.2.1",
     "ts-node": "8.6.2",
     "typescript": "3.7.4"
   },
   "dependencies": {
-    "@angular-devkit/architect": "^0.900.0",
+    "@angular-devkit/architect": "^0.900.3",
     "@angular-devkit/core": ">=8.0.0",
-    "@angular-devkit/schematics": ">=8.0.0",
+    "@angular-devkit/schematics": "^9.0.3",
```

Comment:
could we revert these?

Reply:
Ah didn't want to update the versions here